README.md

```diff
@@ -0,0 +1,78 @@
+# Roadbase
+
+## Introduction
+Roadbase is a POC tool created for crowd-sourcing information about road marks in linked open data structure to enable (local) governments, companies & citizens to use and reuse this data, build new applications and create new roads to safety. This repository includes the tool meant for policy makers to design & request new signs & roadmarkings.
+
+## Deployment
+### Local deployment
+> ❗ Make sure that **[ember-cli](https://cli.emberjs.com/release/)**, **[NodeJS](https://nodejs.org/en/)**, **[Git Bash](https://git-scm.com/downloads)** and **[Docker Desktop](https://www.docker.com/products/docker-desktop)** is installed on your system before continuing.
+
+1. Open a terminal & clone this project to your system.
+    ```bash
+    git clone https://github.com/osoc21/RoadBase.git
+    ```
+2. Navigate to the folder where the project is located.
+    ```bash
+    cd ./RoadBase
+    ```
+    
+3. In the `front-end` directory, install all npm dependencies
+   ```bash
+   cd ./front-end
+   npm install
+   ```
+4. Within the root of the project, run the Docker containers:
+    ```bash
+    docker-compose up
```

Comment:
This still needs the `FRONTEND_URL` and `BACKEND_URL` environment variables set!

For local development:
```
FRONTEND_URL=http://localhost
BACKEND_URL=http://localhost:8000
```

Reply:
Already added it in with the latest push. Thanks for the heads-up!

Reply:
Hahah saw your commit and discord message just after creating this review :p